javascript/adapter.js

```diff
@@ -0,0 +1,76 @@
+import CableReady from 'cable_ready'
+
+class ConsumerAdapter {
+  constructor(consumer) {
+    this.consumer = consumer
+  }
+
+  find_subscriptions(identifier) {
+    throw Error('Not implemented')
+  }
```

Comment:
The idea here is to have an abstract class, not sure what the pattern is for javascript. So would be helpful to hear some feedback!

Reply:
That's an excellent answer. Why on earth is it marked as duplicate. At least I've got some SO powers so I voted for the question to be reopened.